src/api/app/views/webui/request/show.html.haml

```diff
@@ -1,69 +1,116 @@
+                    'data-toggle': 'tab', role: 'tab')
+        %li.nav-item.scrollable-tab-link
+          = link_to('Mentioned Issues', '#mentioned-issues', class: 'nav-link text-nowrap', 'aria-controls': 'mentioned-issues',
+                    'aria-selected': 'false', 'data-toggle': 'tab', role: 'tab')
+      .tab-content.p-4#request-tabs-content
+        .tab-pane.fade.show.p-2.active#overview{ 'aria-labelledby': 'overview-tab', role: 'tabpanel' }
+          = render partial: 'webui/request/beta_show_tabs/overview', locals: { bs_request: @bs_request }
+        .tab-pane.fade.p-2#build-results{ 'aria-labelledby': 'build-results-tab', role: 'tabpanel' }
+          = render partial: 'webui/request/beta_show_tabs/build_results'
+        .tab-pane.fade.p-2#changes{ 'aria-labelledby': 'changes-tab', role: 'tabpanel' }
+          = render partial: 'webui/request/beta_show_tabs/changes'
+        .tab-pane.fade.p-2#mentioned-issues{ 'aria-labelledby': 'mentioned-issues-tab', role: 'tabpanel' }
+          = render partial: 'webui/request/beta_show_tabs/mentioned_issues'
+- else # TODO: remove this else clause once request_show_redesign is rolled out
+  - if @not_full_diff && User.session
+    = render partial: 'webui/shared/truncated_diff_hint', locals: { path: request_show_path(number: @bs_request.number, full_diff: true) }
 
-.row
-  .col-md-8
-    .card.mb-3#comments-list
-      = render partial: 'request_comments', locals: { comments: @comments, bs_request: @bs_request,
-                                                      superseded_requests: @bs_request.superseding.includes(:comments) }
-    = render partial: 'handle_request', locals: { is_target_maintainer: @is_target_maintainer,
-                                                        is_author: @is_author,
-                                                        bs_request: @bs_request,
-                                                        can_handle_request: @can_handle_request,
-                                                        can_add_reviews: @can_add_reviews,
-                                                        my_open_reviews: @my_open_reviews,
-                                                        show_project_maintainer_hint: @show_project_maintainer_hint,
-                                                        package_maintainers: @package_maintainers,
-                                                        actions: @actions }
+  = render partial: 'superseded_by_message', locals: { superseded_by: @bs_request.superseded_by,
+                                                       number: @bs_request.number,
+                                                       diff_to_superseded: @diff_to_superseded,
+                                                       superseding: @bs_request.superseding }
+  .card.mb-3
+    .card-header.d-flex.justify-content-between
+      %h5
+        Overview
+    .card-body
+      .row
+        .col-md-8#overview
```

Comment:
The HAML linter fails now because there are two elements with the `#overview` identifier.

This should not be a problem since one element is in the `if` clause, and the other element is in the `else` clause. :thinking: